dex-it/src/test/scala/com/wavesplatform/it/sync/ReBroadcastUntilConfirmedTestSuite.scala

```diff
@@ -0,0 +1,72 @@
+package com.wavesplatform.it.sync
+
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wavesplatform.dex.domain.order.OrderType
+import com.wavesplatform.dex.it.docker.WavesNodeContainer
+import com.wavesplatform.it.MatcherSuiteBase
+import org.scalatest.EitherValues
+
+class ReBroadcastUntilConfirmedTestSuite extends MatcherSuiteBase with EitherValues {
+
+  // Validator node
+  protected lazy val wavesNode2: WavesNodeContainer =
+    createWavesNode("waves-2", suiteInitialConfig = ConfigFactory.parseString("waves.miner.enable = no") withFallback wavesNodeInitialSuiteConfig)
+
+  override protected def dexInitialSuiteConfig: Config = ConfigFactory
+    .parseString(s"waves.dex.exchange-transaction-broadcast.interval = 10s" +
+      s"| waves.dex.waves-blockchain-client.blockchain-updates-grpc.target = ${wavesNode2.internalIp}:6881")
```

Comment:
For this too: `waves.dex.waves-blockchain-client.grpc.target`

Reply:
Fixed